sql/src/main/java/io/crate/executor/task/DDLTask.java

```diff
@@ -50,6 +51,12 @@ public DDLTask(UUID jobId, DDLStatementDispatcher ddlStatementDispatcher, Generi
         analyzedStatement = genericDDLNode.analyzedStatement();
     }
 
+    public DDLTask(UUID jobId, DDLStatementDispatcher ddlStatementDispatcher, GenericDDLPlan genericDDLPlan) {
```

Comment:
Could probably be reduced to one ctor which takes the analyzedStatement directly